public/sass/custom/kth-klaro.scss

```diff
@@ -203,43 +123,76 @@
         }
       }
 
+      a {
+        color: #036eb8;
+        text-decoration: underline;
+        text-underline-offset: 0.25rem;
+      }
+
+      //Header section
       .cm-header {
         padding: 1.5rem;
 
-        h1 {
-          &.title {
-            font-size: 1.875rem;
-            font-weight: 400;
-            color: $black;
-            line-height: 1.3125;
-            margin-top: 0;
-            margin-bottom: 0.875rem;
+        .hide {
+          svg {
+            stroke: $black;
           }
         }
+
+        .title {
+          font-size: 1.875rem;
+          font-weight: 400;
+          color: $black;
+          line-height: 1.3125;
+          margin-top: 0;
+          margin-bottom: 0.875rem;
+        }
+
+        
       }
 
+      //Body section
       .cm-body {
         padding: 1.5rem;
 
-        ul {
-          &.cm-services {
-            li {
-              &.cm-service {
-                span {
-                  &.cm-list-title {
-                    color: $black;
-                    font-size: 1rem;
-                  }
-                  &.cm-required {
-                    color: $black;
-                    font-size: 1rem;
-                  }
+        .cm-services {
+          .cm-service {
+            span {
+              &.cm-list-title,
+              &.cm-required {
+                color: $black;
+                font-size: 1rem;
+              }
+            }
+
+            .cm-list-label {
+              .slider {
+                background-color: $grey;
+                &:before {
+                  background-color: $white;
                 }
               }
             }
+
+            .cm-list-input {
+              &.required:checked + .cm-list-label .slider {
+                background-color: #3d784a;
```

Comment:
It might be easier to create a local variable for this colour, so that we don't accidentally change it in one place but not the other